src/components/Search/Filter/FiltersContainer.js

```diff
@@ -12,12 +12,12 @@ export default class FiltersContainer extends React.Component<Props, {}> {
   constructor(props) {
     super(props);
     this.config = [
-      {
+      /* {
```

Comment:
Instead of commenting out code, please remove. Can always be referred back to in git history. If this is needed for developer reference, please include a description above.